WordPress/src/main/java/org/wordpress/android/ui/reader/discover/ReaderDiscoverViewModel.kt

```diff
@@ -117,6 +130,20 @@ class ReaderDiscoverViewModel @Inject constructor(
         }
     }
 
+    fun onReblogSiteSelected(siteLocalId: Int) {
+        // TODO malinjir almost identical to ReaderPostCardActionsHandler.handleReblogClicked.
+        //  Consider refactoring when ReaderPostCardActionType is transformed into a sealed class.
+        val state = reblogUseCase.onReblogSiteSelected(siteLocalId, pendingReblogPost)
+        val navigationTarget = reblogUseCase.convertReblogStateToNavigationEvent(state)
+        if (navigationTarget != null) {
+            _navigationEvents.postValue(Event(navigationTarget))
+        } else {
+            // TODO malinjir show toast R.string.reader_reblog_error
```

Comment:
This is handled in https://github.com/wordpress-mobile/WordPress-Android/pull/12358